schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth date,
     escape_attempts integer,
     neutered boolean,
-    weight_kg decimal
+    weight_kg decimal,
+    species varchar(100)
```

Comment:
- [ ] Kindly, it would be better to write a query to modify the `animals` table as suggested in the project requirements, you may try to use the `ALTER TABLE` to achieve this 👍 

![image](https://user-images.githubusercontent.com/102757126/229293712-2c0eeb9c-d4f5-41d1-8b6b-b57b50b6faaa.png)